response_operations_ui/templates/survey.html

```diff
@@ -61,6 +63,9 @@ <h2 class="saturn u-pt-m">Collection exercises for {{survey.shortName}}</h2>
               <td class="table--cell tbl-ce-respondent-info" name="tbl-ce-shown-as">
                 {{exercise.userDescription}}
               </td>
+              <td class="table--cell tbl-ce-status" name="tbl-ce-status">
+                {{exercise.state}}
```

Comment:
You will need to map the the states to a more user readable state:
INIT --> Created
PENDING --> Scheduled
VALIDATED, EXECUTED, PUBLISHED --> Ready for Review, Ready for Live, Live

I would double check this with both the rad team and the devs developing the backend state transition.

Reply:
Yeah i've spoken with Rachel, looks like we need to have a session around the mapping of the backend to frontend states, as these might be incorrect now